src/mobile/components/Navigator/Navigator.tsx

```diff
@@ -348,11 +384,18 @@ export default ({ toggle }: NavigatorProps) => {
                           title: `Rename "${storage.name}" storage`,
                           message: t('storage.renameMessage'),
                           iconType: DialogIconTypes.Question,
-                          defaultValue: storage.name,
+                          defaultValue: {
+                            folderPath: storage.name,
+                          },
                           submitButtonLabel: t('storage.rename'),
-                          onClose: async (value: string | null) => {
-                            if (value == null) return
-                            await renameStorage(storage.id, value)
+                          onClose: (value: PromptDialogValues | null) => {
```

Comment:
Any reason why you remove `async` from this function?

Reply:
I didn't think I made that change. Problem solved!